cmake/DARTFindDependencies.cmake

```diff
@@ -28,7 +28,11 @@ if(ASSIMP_FOUND)
   # Check for missing symbols in ASSIMP (see #451)
   include(CheckCXXSourceCompiles)
   set(CMAKE_REQUIRED_DEFINITIONS "")
-  set(CMAKE_REQUIRED_FLAGS "")
+  if (NOT ASSIMP_VERSION VERSION_LESS 3.3.0)
```

Comment:
We could have `AND NOT MSVC` here, because MSVC does not know the `-std=c++11` flag.

Then again, even if we pass the flag in for MSVC, I think the compiler will just ignore it and proceed as normal, so it doesn't really matter.

Reply:
Good catch! Yeah, it would work with the flag, but suppressing an unnecessary warning is worth enough. :smile: